Mobile/ShopLens/Droid/Activities/CameraActivity.cs

```diff
@@ -218,11 +222,23 @@ private void RecogniseImage(Uri uri)
                         QueueMode.Flush,
                         null,
                         null);
-
+                    new ErrorDialogCreator(this, "Shopping cart", "Would you like to add to thid product to your shopping cart?", "Yes", "No", 
```

Comment:
Maybe the message should be a variable or an App.config constant? Also, grammar errors.